system_files/usr/share/ublue-os/just/50-devmode.just

```diff
@@ -0,0 +1,50 @@
+
+alias devmode := change-edition
+# Allows selection of Bluefin LTS editions
+change-edition:
+    #!/bin/bash
+
+    # Function to display the menu and get user selection using gum
+    select_edition() {
+    edition=$(gum choose --header "Please select a edition:" "HWE" "GDX" "DX" "Base")
```

Comment:
Valid options after the user chooses rebase should be: 

date, gdx, lts, lts-hwe

dx isn't an option, that's a separate command that just turns dx on and off. It's a separate command from the rebase helper.